src/utils/reqwest.rs

```diff
@@ -12,6 +15,21 @@ pub fn default_retry_policy() -> ExponentialBackoff {
     ExponentialBackoff::builder().build_with_max_retries(3)
 }
 
+fn auth_middleware() -> AuthenticationMiddleware {
+    if let Ok(auth_file) = std::env::var("RATTLER_AUTH_FILE") {
```

Comment:
Should this file location also be part of the global config?

Reply:
I made it so! :)

Reply:
Could you document it aswell